src/napari_epitools/analysis/projection.py

```diff
@@ -120,7 +121,9 @@ def calculate_projection(
     t_size, z_size, y_size, x_size = input_image.shape
     smoothed_imstack = _smooth(input_image.astype(np.float64), smoothing_radius)
 
-    t_interp = np.zeros((t_size, y_size, x_size))  # remove the z-axis when projecting
+    t_interp = np.zeros(
+        (t_size, SINGLE_SLICE, y_size, x_size)
+    )  # single slice in the Z dimension
```

Comment:
More readable
```suggestion
    # single slice in the Z dimension
    t_interp = np.zeros((t_size, SINGLE_SLICE, y_size, x_size))
```

Reply:
I've moved the commend to the top of the block and expanded it a bit